### ISSUE TITLE
GET /errors returns 500 error

### ISSUE THREAD

**Issue by blais** [beancount/beancount#85]
**[Original report](https://bitbucket.org/blais/beancount/issue/85) by Zhuoyun Wei (Bitbucket: [wzyboy](https://bitbucket.org/wzyboy), GitHub: [wzyboy](https://github.com/wzyboy)).**

----------------------------------------

In bean-web, whenever I navigate to "/errors" page, I get an HTTP 500 error.

The Traceback:

```
#!python

Traceback (most recent call last):
  File "/home/wzyboy/git/beancount/BEANCOUNT/bin/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/home/wzyboy/git/beancount/BEANCOUNT/bin/bottle.py", line 1732, in wrapper
    rv = callback(*a, **ka)
  File "/home/wzyboy/git/beancount/BEANCOUNT/lib/python3.5/site-packages/beancount/web/web.py", line 962, in wrapper
    return callback(*posargs, **kwargs)
  File "/home/wzyboy/git/beancount/BEANCOUNT/lib/python3.5/site-packages/beancount/web/web.py", line 323, in errors
    contents=render_report(misc_reports.ErrorReport, [], leaf_only=False))
  File "/home/wzyboy/git/beancount/BEANCOUNT/lib/python3.5/site-packages/beancount/web/web.py", line 159, in render_report
    report_.render_htmldiv(entries, app.errors, app.options, oss)
  File "/home/wzyboy/git/beancount/BEANCOUNT/lib/python3.5/site-packages/beancount/reports/misc_reports.py", line 51, in render_htmldiv
    self.formatter.render_source(error.source),
  File "/home/wzyboy/git/beancount/BEANCOUNT/lib/python3.5/site-packages/beancount/web/web.py", line 132, in render_source
    return '{}#{}'.format(app.get_url('source'), source.lineno)
AttributeError: 'dict' object has no attribute 'lineno'
```

**Comment by blais** [comment]
**Original comment by Alok Parlikar (Bitbucket: [happyalu](https://bitbucket.org/happyalu), GitHub: [happyalu](https://github.com/happyalu)).**

----------------------------------------

https://bitbucket.org/blais/beancount/src/1a9e288fabb60f1d56755ba284caf235a574fc41/src/python/beancount/web/web.py?at=default&fileviewer=file-view-default#web.py-132 should have source['lineno'] I think.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #85: Reference to '.lineno' hadn't been converted in bean-web somehow.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Thanks Zhuoyun & Alok,
